javascript/javascript1/week4/Voice assistant/voice-assistant.js

```diff
@@ -0,0 +1,28 @@
+
+const userInformation = {
+    name: '',
+    toDoList: [],
+    day: '',
+    favoriteDish: '',
+    myCalendar: []
+};
+
+  //Introducing myself
+  function getReply(command) {
+    if(command.toString().toLowerCase() === 'hello my name is Benjamin'.toLowerCase()){
```

Comment:
nice start,

you should look at using the startswith and endswith functions 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith

```
function getReply(command) {
   if (command.startswith("Hello! my name is")) {
   // ....
}
```
that will make it easier to know what commands to handle, for example if it starts with `Add` and ends with `todo`